tests/src/org.opengauss/opengauss-jdbc/3.1.0-og/src/test/java/org_opengauss/opengauss_jdbc/Opengauss_jdbcTest.java

```diff
@@ -0,0 +1,254 @@
+                assertThat(resultSet.next()).isTrue();
+                for (int i = 1; i <= 7; i++) {
+                    System.out.printf("column %d: %s%n", i, resultSet.getObject(i));
+                }
+            }
+        }
+    }
+
+    @Test
+    void simpleDatatypes() throws Exception {
+        try (Connection connection = openConnection()) {
+            connection.setAutoCommit(false);
+            // OpenGauss 3.1.0 does not support xml type
+            connection.prepareStatement("""
+                    CREATE TABLE simple_datatypes
+                    (
+                        b1 bigint,
+                        b2 bit(1),
+                        b3 boolean,
+                        b4 bytea,
+                        c1 character(1),
+                        c2 character varying,
+                        d1 date,
+                        d2 double precision,
+                        i1 integer,
+                        j1 json,
+                        j2 jsonb,
+                        r1 real,
+                        s1 smallint,
+                        t1 text,
+                        t2 time,
+                        t3 time with time zone,
+                        t4 timestamp,
+                        t5 timestamp with time zone,
+                        u1 uuid,
+                        x1 text,
+                        n1 integer
+                    )
+                    """).execute();
+            PreparedStatement statement = connection.prepareStatement("""
+                    INSERT INTO simple_datatypes
+                    VALUES (?, CAST(? AS bit), ?, ?, ?, ?, ?, ?, ?, CAST(? AS json), CAST(? AS jsonb), ?, ?, ?, ?, ?, ?, ?, ?,
+                            CAST(? AS text), ?)
+                    """);
+            statement.setLong(1, Long.MAX_VALUE);
+            statement.setString(2, "1");
+            statement.setBoolean(3, true);
+            statement.setObject(4, "some-bytes".getBytes(StandardCharsets.UTF_8));
+            statement.setString(5, "c");
+            statement.setString(6, "some-string");
+            statement.setDate(7, Date.valueOf(LocalDate.of(2020, 1, 2)));
+            statement.setDouble(8, Math.PI);
+            statement.setInt(9, 42);
+            statement.setString(10, "{ \"foo\": \"bar\"}");
+            statement.setString(11, "{ \"foo\": \"bar\"}");
+            statement.setFloat(12, (float) Math.PI);
+            statement.setInt(13, 5);
+            statement.setString(14, "some-text");
+            statement.setObject(15, LocalTime.of(23, 59));
+            // org.opengauss:opengauss-jdbc:3.1.0-og does not support java.time.OffsetTime
+            statement.setObject(16, LocalTime.of(23, 59, 0, 0));
+            statement.setObject(17, LocalDateTime.of(2020, 1, 2, 23, 59, 0));
+            statement.setObject(18, OffsetDateTime.of(2020, 1, 2, 23, 59, 0, 0, ZoneOffset.ofHours(3)));
```

Comment:
- @edisonxiang I don't mean to bother, but some countries can't log into Gitee to report issues of opengauss-jdbc. Is it intentional that opengauss-jdbc does not support JSR-310?